Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -821,44 +861,48 @@ void AsyncScrollingCoordinator::scrollableAreaScrollbarLayerDidChange(Scrollable
         scrollableArea.horizontalScrollbarLayerDidChange();
 }
 
-ScrollingNodeID AsyncScrollingCoordinator::createNode(ScrollingNodeType nodeType, ScrollingNodeID newNodeID)
+ScrollingNodeID AsyncScrollingCoordinator::createNode(FrameIdentifier rootID, ScrollingNodeType nodeType, ScrollingNodeID newNodeID)
```

Comment:
`rootID` -> `rootFrameID` (prefer variable names with the least amount of ambiguity).

Reply:
Fixed.